index.js

```diff
@@ -2212,7 +2240,7 @@ if (!window.console) {
      * var cb = ClearBlade.Messaging({}, callback);
      * cb.disconnect()//why leave so soon :(
      */
-    messaging.disconnect = function() {
+    messaging.disconnect = function () {
```

Comment:
@clarkbynum do you know if a disconnect automatically clear all subscriptions? if so, i think we need to clear the callbacks structure we are storing here.

Reply:
Not sure. @seschwartz8 will you look into the disconnect method in the Paho lib to see if subscriptions are cleared?

Reply:
It appears that because we have set "cleanSession: true" I believe the subscriptions will be cleared on disconnect. So I can add a line that resets messageCallbacks = {} in this disconnect method